launcher/main.cpp

```diff
@@ -35,7 +35,7 @@ int main(int argc, char * argv[])
 
 	MainWindow mainWindow;
 	mainWindow.show();
-	result = vcmilauncher.exec();
+	result = QApplication::exec();
```

Comment:
now there'll be warning "variable vcmilauncher is unused"

Reply:
Right, will drop the commit.